### PR TITLE
Implement role filtering on magic commands

### PR DESCRIPTION
**Pull request by 2Cubed**
Completes #109.

### CODE REVIEW

File: cactusbot/commands/magic/alias.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ class Alias(Command):
 
     COMMAND = "alias"
 
-    @Command.command()
+    @Command.command(role="moderator")
```
Comment (Innectic) [inline cactusbot/commands/magic/alias.py]: Maybe we should make these integer based?

Comment (2Cubed) [inline cactusbot/commands/magic/alias.py]: [They are integer-based](https://github.com/CactusDev/CactusBot/pull/134/files/5a5e9421a5b472f6fc8db009c7de3d68de6ab991#diff-fa8091e5f9fd54753ebcf602d35099faR6)... but you also have the option to use strings, just for readability/clarity. (They're the preferred method for the decorators.)

File: cactusbot/commands/command.py [hunk 2]
```diff
@@ -3,6 +3,14 @@
 import inspect
 import re
 
+ROLES = {
```
Comment (Innectic) [inline cactusbot/commands/command.py]: Why did you skip 3 here?

Comment (2Cubed) [inline cactusbot/commands/command.py]: Good question - basically, I wanted to leave it open for a potential "new" thing, be that regulars, or CactusDev subscribers, or... whatever. Also, 5 is a nice number. ;P

**Pull request merged**